apps/dashr-financial-report/assets/FinancialFunctions.R

```diff
@@ -0,0 +1,130 @@
+# Install a package that is necessary and has not been installed, or load one that is.
+
+usePackage <- function(p) 

```

Comment:
@HammadTheOne Where did this function come from?

Reply:
🔪, remnant of the installation process before knowing about `init.R` file.